Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabSystemComponent.cpp

```diff
@@ -753,6 +754,80 @@ namespace AzToolsFramework
             }
         }
 
+        bool PrefabSystemComponent::AreDirtyTemplatesPresent(TemplateId templateId)
```

Comment:
maybe: AreDirtyChildTemplatesPresent(TemplateId rootTemplateId)

Reply:
My concern was that this function doesn't really tell that you are going to find dirty templates within the given template argument, rather than dirty templates in general. With the naming change and variable you can tell better that you are performing the search within the search argument

Reply:
Changed the names to rootTemplateId to indicate that this is the starting template id. ReferencedTemplates is still not obvious imo. With the change of parameter to rootTemplateId and the function comments, I think the intention of this function is much clearer now.